### PR TITLE
Run goferd optimized by default

### PR DESCRIPTION
**Pull request by seandst**

### CODE REVIEW

File: docs/sphinx/installation.rst [hunk 1]
```diff
@@ -14,6 +14,13 @@ Gofer is packaged into RPMs for Linux.  These packages are as follows:
 Depending on system capabilities, the *gofer* package registers goferd
 with systemd or upstart service managers.
 
+python optimizations
+--------------------
+
+By default, gofer runs python with optimizations enabled. If you want to disable those
```
Comment (jortel) [inline docs/sphinx/installation.rst]: Might be better to specifically say "goferd runs" ...

File: gofer.spec [hunk 2]
```diff
@@ -92,6 +94,7 @@ rm -rf %{buildroot}
 %else
 %attr(755,root,root) %{_sysconfdir}/init.d/%{name}d
 %endif
+%attr(755,root,root) %{_sysconfdir}/sysconfig/%{name}d
```
Comment (jortel) [inline gofer.spec]: Should be 644

Comment (seandst) [inline gofer.spec]: That was delicious copypasta, good catch...

File: gofer.spec [hunk 3]
```diff
@@ -73,6 +74,7 @@ cp usr/lib/systemd/system/* %{buildroot}/%{_unitdir}
 %else
 cp etc/init.d/%{name}d %{buildroot}/%{_sysconfdir}/init.d
 %endif
+cp etc/sysconfig/%{name}d %{buildroot}/%{_sysconfdir}/sysconfig
```
Comment (jortel) [inline gofer.spec]: Suggest this be moved to just after line 68 so it's grouped with other configuration.

Comment (seandst) [inline gofer.spec]: I missed the noreplace on the sysconfig file, too...fixing both now

**Comment by jortel** [comment]
Unit tests are clean.  Running functional tests now.

**Comment by jortel** [comment]
Looks good.  Thanks for the PR!

**Pull request merged**